src/mlpack/methods/ann/activation_functions/quadratic_function.hpp

```diff
@@ -0,0 +1,81 @@
+/**
+ * @file methods/ann/activation_functions/quadratic_function.hpp
+ * @author Himanshu Pathak
+ *
+ * Definition and implementation of quadratic function.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_ACTIVATION_FUNCTIONS_QUAD_FUNCTION_HPP
+#define MLPACK_METHODS_ANN_ACTIVATION_FUNCTIONS_QUAD_FUNCTION_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace ann /** Artificial Neural Network. */ {
+
+/**
+ * The Quadratic function, defined by
+ *
+ * @f{eqnarray*}{
+ * f(x) = x^2 \\
+ * f'(x) = 2*x \\
+ * @f}
+ */
+class QuadFunction
```

Comment:
Not sure should we call this `QuadraticFunction`?

Reply:
Done.